tests/test_validators.py

```diff
@@ -65,3 +77,28 @@ def test_register(self):
 
         validator = Validator.build({"name": "is-the-answer"})
         assert isinstance(validator, self.IsTheAnswerValidator)
+
+
+class TestGreaterThan(TestCase):
+    def test_is_above_threshold(self):
```

Comment:
Le nom des tests devrait décrire le comportement attendu et réutiliser le même langage.
Le but du test c'est à la fois de vérifier que le code fonctionne comme attendu mais aussi de documenter le comportement de ta classe, ta fonctionnalité.
```suggestion
    def test_it_successfuly_returns_the_value_when_greater_than_threshold(self):
```